repondeur/zam_repondeur/templates/lecture.html

```diff
@@ -0,0 +1,45 @@
+{% extends "_base.html" %}
+
+
+{% block breadcrumbs %}
+    <li class="breadcrumb-item"><a href="{{ request.route_url('lectures_list') }}">Lectures</a></li>
+    <li class="breadcrumb-item active" aria-current="page">{{ lecture }}</li>
+{% endblock %}
+
+
+{% block body %}
+    <h2>{{ lecture }}</h2>
+
+    <div class="card">
+        <div class="card-body">
+            <div class="card-title">
+                <h4>Récupérer les amendements</h4>
+            </div>
+            <div class="card-text">
+            {% if lecture.chambre == 'senat' %}
+                <p>Récupérer un tableau de tous les amendements déposés, triés par ordre de passage, depuis le site du Sénat.</p>
+                <a class="btn btn-primary" href="{{ request.route_url('amendements_csv', chambre=lecture.chambre, session=lecture.session, num_texte=lecture.num_texte) }}"><i class="fa fa-table"></i>Télécharger (CSV)</a>
+                <a class="btn btn-primary" href="{{ request.route_url('amendements_xlsx', chambre=lecture.chambre, session=lecture.session, num_texte=lecture.num_texte) }}"><i class="fa fa-file-excel"></i>Télécharger (Excel)</a>
+            {% elif lecture.chambre == 'assemblee' %}
+                <p>La récupération des amendements depuis le site de l'Assemblée nationale n'est pas encore disponible. 🚧</p>
+            {% endif %}
+            </div>
+        </div>
+    </div>
+
+{% endblock %}
+
+
+{% block header %}
```

Comment:
I used to put the block in the same order as in the base template.

Reply:
Consistency FTW.

Reply:
Fixed in cce7ed7